cyoa/main/main.go

```diff
@@ -35,20 +38,18 @@ func handlerHttp(w http.ResponseWriter, r *http.Request) {
 
 	t, _ := template.ParseFiles("template.html")
 
-	val := reflect.ValueOf(story)
-	typ := reflect.TypeOf(story)
-	var retVal string
+	retVal, ok := story[expectedArc]
 
-	for i := 0; i < val.NumField(); i++ {
-		if typ.Field(i).Name == expectedArc {
-			fmt.Println(val.Field(i))
-			retVal = val.Field(i).String()
-		}
+	if !ok {
+		fmt.Printf("path %s not found in the data source\n", expectedArc)
+		return
 	}
 
 	err = t.Execute(w, retVal)
 
 	if err != nil {
+		fmt.Println("whaa")
```

Comment:
I forgot to remove this caveman debugging stuff - I suggest on next session we spend some time learning and writing how to elegantly handle errors so that stack traces are nice & understandable. WDYT ?

Reply:
I generally love this concept that you have to take care of error handling every time there is error prone situation.
But every time I actually need to this I am to lazy to do it right and hate for myself for doing soo 😄  
Lets fix that love hate situation in next meeting 💘

Reply:
I substituted the calls to `panic` for something else in [this PR](https://github.com/hs3city/goexplorers/pull/9/files)